pallets/auction/src/lib.rs

```diff
@@ -493,6 +496,20 @@ pub mod pallet {
 
 							Self::deposit_event(Event::BuyNowFinalised(auction_id, from, value));
 						}
+						ItemId::UndeployedLandBlock(undeployed_land_block_id) => {
```

Comment:
I think listing fee should be applied as well even for undeployed land block so protocol is consistent.

Reply:
Maybe network fee when it's fully implemented.